README.md

```diff
@@ -87,6 +87,17 @@ See
 [ESLint documentation](http://eslint.org/docs/user-guide/configuring#extending-configuration-files)
 for more information about extending configuration files.
 
+### All
```

Comment:
add a caveat about this not being subject to semver, so use with caution, blahblah 🙂

Reply:
Done